install-3scale-rwo/apimanager.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: apps.3scale.net/v1alpha1
+kind: APIManager
+metadata:
+  name: "3scale-amp"
+  namespace: "3scale-rwo"
```

Comment:
namespace is not required and if they are using a different namespace this will make it look like the apimanager disappeared. I would remove this line.

Reply:
This does not belong here. Removed it